src/rmm.cpp

```diff
@@ -54,32 +59,29 @@ const char * rmmGetErrorString(rmmError_t errcode) {
 // Initialize memory manager state and storage.
 rmmError_t rmmInitialize(rmmOptions_t *options)
 {
+
     rmm::Manager::getInstance().initialize(options);
+    rmm::mr::device_memory_resource * memory_resource = rmm::mr::detail::cuda_resource();
```

Comment:
Use the default resource. 
```suggestion
    rmm::mr::device_memory_resource * memory_resource = rmm::mr::detail::initial_resource();
```